### PR TITLE
Designed UI

### PR DESCRIPTION
**Pull request by VICTOR-CHUKWU**
Created component folder
created calculator js
designed UI

### CODE REVIEW

**Review by Elbie-em — COMMENTED** [review]
## STATUS:  INVALID CRR 🔴 ❌

Hi @VICTOR-CHUKWU, while you made a great effort in this project, unfortunately, I cannot proceed to review your code.

You have submitted a project that does not use [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md). 
Kindly create a development branch and this should be the base of comparison for this PR.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Kindly submit another review request once done and please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

> Elbie Moonga (Part-time  Code Reviewer)

File: src/App.js [hunk 1]
```diff
@@ -1,27 +1,10 @@
...
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div>
+      <Calculator />
```
Comment (arslanbisharat) [inline src/App.js]: - [ ] APP should be class-based. Please have a look at the project requirements.

File: src/component/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+/* eslint-disable react/prefer-stateless-function */
```
Comment (arslanbisharat) [inline src/component/Calculator.js:1]: - [ ] Let's not disable eslint rules. These are here to write clean code.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by dasileker — APPROVED** [review]
## **Hi @VICTOR-CHUKWU,**


## **Congratulations! 🎉 Project Approved**

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀
 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**